cif_file_ingester/parse_cif_pmg.py

```diff
@@ -1,4 +1,5 @@
 from pypif.obj import *
+import re
 import ase.io
 from pymatgen import Structure
 from pymatgen.io.ase import AseAtomsAdaptor
```

Comment:
What exactly is your proposal @ckborg ?

Reply:
@maxhutch Define system = None so that the expect block doesn't return None and cause the rest of the parsing to stop. We can still get info from the text. I think pymatgen doesn't like this file due to it's encoding so maybe we want to handle file the opening separate and see if pymatgen has Structure_from_string() (just thought of this now).

Reply:
How about something like this?
<img width="556" alt="screen shot 2018-11-14 at 2 30 47 pm" src="https://user-images.githubusercontent.com/35743526/48517051-f1999200-e819-11e8-955a-59dbe414e398.png">

Reply:
That works!